src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -173,33 +173,31 @@ define(function (require, exports, module) {
             stat(path, callback);
         });
     }
-
-    function rename(oldPath, newPath, callback) {
+   function rename(oldPath, newPath, callback) {
         oldPath = decodePath(oldPath);
         newPath = decodePath(newPath);
 
         function updateBlobURL(err) {
             if(err) {
                 return callback(_mapError(err));
             }
-
+        fs.refresh(callback);
```

Comment:
The `refresh()` method doesn't live on the `fs` object.  It's part of the `FileSystemCache` module I linked you to previously.  You'll have to `require` it up on lines 7-15 with the other modules before you can use it.